src/features/click-to-play.js

```diff
@@ -364,22 +367,6 @@ class DuckWidget {
     }
 }
 
-/**
- * Initialise the Click to Load feature, once the necessary details have been
- * returned by the platform.
- * @returns {Promise}
- */
-async function initCTL () {
```

Comment:
Having both `init` and `initCTL` was confusing. Especially with these changes, where `initCTL` needed to do even less work. I thought it would be better to remove this therefore, but I'm still hoping to move more of the logic into `init` in the future.